src/components/colorPicker/colorPicker.api.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "ColorPicker",
+  "category": "form",
+  "description": "A picker component for color selection",
+  "notes": "This is a screen width component",
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/ColorPickerScreen.tsx",
+  "images": [
+    "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/ColorPicker/ColorPicker.gif?raw=true"
+  ],
+  "props": [
+    {
+      "name": "colors",
+      "type": "string[]",
+      "description": "Array of colors for the picker's color palette (hex values)"
+    },
+    {"name": "value", "type": "string", "description": "The value of the selected swatch"},
+    {
+      "name": "animatedIndex",
+      "type": "number",
+      "description": "The index of the item to animate at first render",
+      "note": "Default is last"
```

Comment:
looks like `default` is a better key here

Reply:
No. "last" is not a value, it means the last item in the array.